crates/bindings-csharp/Runtime/Internal/ITable.cs

```diff
@@ -2,121 +2,120 @@ namespace SpacetimeDB.Internal;
 
 using SpacetimeDB.BSATN;
 
-public interface ITable<T> : IStructuralReadWrite
-    where T : ITable<T>, new()
+internal abstract class RawTableIterBase<T>
+    where T : IStructuralReadWrite, new()
 {
-    // These are the methods that codegen needs to implement.
-    static abstract IEnumerable<TableDesc> MakeTableDesc(ITypeRegistrar registrar);
-
-    internal abstract class RawTableIterBase
+    public sealed class Enumerator(FFI.RowIter handle) : IDisposable
     {
-        public sealed class Enumerator(FFI.RowIter handle) : IDisposable
-        {
-            byte[] buffer = new byte[0x20_000];
-            public byte[] Current { get; private set; } = [];
+        byte[] buffer = new byte[0x20_000];
```

Comment:
How often is one of these created? I've heard wasm memory allocators are often pretty slow, but I don't know wrt C#. It would be nice to poke at a C# module with a memory profiler, but I'm not sure if any such thing exists for wasm...

Reply:
Yeah few people were concerned about it. I tried couple of approaches to replace it with something more sensible, but it didn't make any noticeable improvement to benchmarks - I assume having memory reused within Wasm instance helps.

Might "optimise" it anyway in the future, but definitely out of scope of this PR.